yarn.lock

```diff
@@ -2094,6 +2104,11 @@ browserslist@^4.0.0, browserslist@^4.12.0, browserslist@^4.14.5, browserslist@^4
     escalade "^3.1.1"
     node-releases "^1.1.71"
 
+bs-custom-file-input@^1.3.4:
```

Comment:
This is a BS4 package, are there BS5 alternatives or will we need to make something custom to handle what this handles?